assets/js/events.js

```diff
@@ -8,14 +8,21 @@ const eventsScript = (function(jQuery) {
     const today = new Date().toISOString().slice(0, 10).replace(/-/g, '');
 
     const showFeaturedEvents = function() {
-        const events = jQuery('#featured-events .event-card');
-
-        const sortedEvents = events.sort(function(a, b) {
-            return jQuery(b).data('expiration') - jQuery(a).data('expiration');
+        const sortedEventsAscending = jQuery('#featured-events .event-card').sort(function(a, b) {
+            return jQuery(a).data('expiration') - jQuery(b).data('expiration');
+        });
+        
+        const upcomingEvents = sortedEventsAscending.filter(function() {
+            return jQuery(this).data('expiration') >= today;
         });
 
+        if(upcomingEvents.length === 0) {
+            jQuery('#featured-events-container').addClass(CLASS_HIDDEN);
+            return;
+        };
+
         let validEventCount = 0;
-        sortedEvents.each(function() {
+        sortedEventsAscending.each(function() {
```

Comment:
Should not be descending?

Reply:
Yes, for the featured events on home page, it would show the earliest upcoming events (i.e the ones in August) to promote them first.

With descending, it would show the last ones instead which would be september and not promote the ones happening first